presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -8272,6 +8272,29 @@ public void testRightJoinWithEmptyProbeSide()
                 "WITH small_part AS (SELECT * FROM part WHERE name = 'a') SELECT lineitem.orderkey FROM small_part RIGHT JOIN lineitem ON  small_part.partkey = lineitem.partkey");
     }
 
+    @Test
+    public void testLastValueIgnoreNulls()
+    {
+        assertQuery("WITH T AS (" +
```

Comment:
Nits: new line for argument.
```
    assertQuery(
        "WITH T ...
```

Reply:
Done. Don't know why 'Reformat Code' doesn't do it automatically though

Reply:
I don't think our intelij style setup really conforms with the actual style we are following :(